keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -134,107 +137,94 @@ When a workload (app pod, deployment, configs) is moved to another cluster, as l
-1. `labels`: added by COSI. The "cosi.io/provisioner" key's value is the provisioner name. The "cosi.io/bucket-prefix" key's value is the `BucketRequest.bucketPrefix` value when specified. Characters that do not adhere to [Kubernetes label conventions](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) will be converted to ‘-’.
-1. `finalizers`: added by COSI to defer `Bucket` deletion until backend deletion ops succeed. Implementation may add one finalizer for the BR and one for the BA.
-1. `provisioner`: (optional) The provisioner field as defined in the `BucketClass`. If empty then there is no driver/sidecar, thus the admin had to create the `Bucket` and `BucketAccess` instances.
-1. `retentionPolicy`: Prescribes the outcome when the `BucketRequest` is deleted.
-   - _Retain_: (default) the `Bucket` and its data are preserved. The `Bucket` can potentially be reused.
-   - _Delete_: the bucket and its contents are expected to be deleted by the backend store.
-A `Bucket` is not deleted if it is bound to a `BucketRequest`.
-1. `anonymousAccessMode`: a string specifying *uncredentialed* access to the Bucket.  This is applicable to cases where the bucket objects are intended to be publicly readable and/or writable. One of:
-   - "private": Default, disallow uncredentialed access to the backend storage.
-   - "publicReadOnly": Read only, uncredentialed users can call ListBucket and GetObject.
-   - "publicWriteOnly": Write only, uncredentialed users can only call PutObject.
-   - "publicReadWrite": Read/Write, same as `ro` with the addition of PutObject being allowed.
-   > Note: `anonymousAccessMode` is intended for workloads to consume the bucket out of band, i.e. COSI does not provide a automated mechanism for a k8s workload to consume the bucket anonymously
+1. `name`: when created by COSI, the `Bucket` name is generated in one of these formats: _"br-"+uuid_ (if BR.prefix is empty), or BR.prefix+"-"+uuid (if prefix is supplied). If an admin creates a `Bucket`, as is necessary for static brownfield access, any unique name is allowed.
+1. `labels`: added by COSI. The "objectstorage.k8s.io/provisioner" key's value is the provisioner name. The "objectstorage.k8s.io/bucket-prefix" key's value is the `BucketRequest.bucketPrefix` value when specified. Characters that do not adhere to [Kubernetes label conventions](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) will be converted to ‘-’.
+1. `finalizers`: added by COSI to defer `Bucket` deletion until backend deletion succeeds. Implementation may add one finalizer for the BR and one for each BA pointing to this `Bucket`. **Note**: a `BucketAccess` object has one finalizer per accessing pod. So there is no access to a bucket when all `Bucket` finalizers have been removed.
```

Comment:
I think Nicolas commented this below, but this specific implementation is still not finalized. I believe this is not mentioned in the finalizer section of the KEP as well.

Reply:
> this specific implementation is still not **finalized**

(pun intended, I assume!)
It is true the all mentions in the kep of finalizers, their name, when added, when removed, etc is merely to convey that they _could_work this way to control deletions, but details are left to the implementors.

Reply:
Yes, that is correct